src/main/java/com/sorting/Selection.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * File: Selection.java
+ * Date: 10 Dec, 2015
+ * Author: Tarun Jain
+ * -----------------------------------------------
+ * Sorting is the process of rearranging a sequence of objects so as to put them 
+ * in some logical order. The selection sort algorithm sorts an array by repeatedly 
+ * finding the minimum element (considering ascending order) from unsorted part and 
+ * putting it at the beginning. It exchange one for every pass through the list. 
+ * 
+ *  *  Features:
+ *  
+ *  1. Selection Sort is implemented as an array of Comparable Objects which sorts
```

Comment:
Change the point #1 as Selection Sort algorithm takes an array of Comparable objects and sorts them based on their natural order defined by Comparable interface.

Add another point 2 as: In case if client wants to sort the objects in some other order than the natural order of objects, then client can call the overloaded method Selection.sort(array, comparator) and pass the comparator object which gives the customized ordering.